### PR TITLE
USHIFT-421: Update manifests to more closely align with OCP

### PR DESCRIPTION
**Pull request by fzdarsky**
Small updates to the rebase script and the produced assets to more closely match OCP's deployments.

Signed-off-by: Frank A. Zdarsky <fzdarsky@redhat.com>

Closes [USHIFT-421](https://issues.redhat.com//browse/USHIFT-421)

### CODE REVIEW

**Comment by dhellmann** [comment]
This looks OK to me. I'll hold off the lgtm until we sort out CI to avoid unnecessary retests.

File: pkg/assets/bindata.go [hunk 1]
```diff
@@ -1838,6 +1838,9 @@ data:
     .:5353 {
         bufsize 512
         errors
+        log . {
```
Comment (pmtk) [inline pkg/assets/bindata.go:1841]: Please remove bindata.go from your PR

**Review by pmtk — CHANGES_REQUESTED** [review]

File: pkg/assets/bindata.go (hunk 1 above)
Comment (fzdarsky) [reply pkg/assets/bindata.go]: done

**Pull request merged**